src/NetscriptFunctions/Corporation.ts

```diff
@@ -753,6 +753,14 @@ export function NetscriptCorporation(
           "Research & Development": office.employeeProd[EmployeePositions.RandD],
           Training: office.employeeProd[EmployeePositions.Training],
         },
+        employeeJobs: {
+          Operations: office.employeeJobs[EmployeePositions.Operations],
+          Engineer: office.employeeJobs[EmployeePositions.Engineer],
+          Business: office.employeeJobs[EmployeePositions.Business],
+          Management: office.employeeJobs[EmployeePositions.Management],
+          "Research & Development": office.employeeJobs[EmployeePositions.RandD],
+          Training: office.employeeJobs[EmployeePositions.Training],
```

Comment:
we still need to expose total

Reply:
Total is not exposed in the EmployeeJobs interface currently, and as such the employeeProd field above doesn't expose it either. Should they both?